Sources/IRGen/Expression/IRExpression.swift

```diff
@@ -8,6 +8,20 @@
 import AST
 import Lexer
 
+struct GeneratedCode {
```

Comment:
Is this specific to expressions or is this for general application within the code generator? 

I would prefer a name such as `Fragment` or if specific to expressions, `ExpressionFragment`. This way we know that this is partial and not necessarily complete code. Definitely be specific that this is for expressions.